crates/ln-dlc-node/src/ldk_node_wallet.rs

```diff
@@ -90,6 +93,8 @@ where
             "Finished on-chain sync",
         );
 
+        self.locked_outpoints.lock().clear();
```

Comment:
🤔 is it safe to clear the utxos here? or do we even have to wait for the locked utxo to get a confirmation, before we can safely remove it from the `locked_outputs`?

Reply:
I believe it is because during sync with on-chain we should catch-up with all transactions which are in the mempool